### PR TITLE
feat: Add defined schema docs

### PR DESCRIPTION
**Pull request by Moumouls**

### CODE REVIEW

**Comment by Moumouls** [comment]
@mtrezza here the final PR updated with lockfile V2

File: _includes/defined-schema/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    name: { type: "String", required: true },
+    location: { type: "GeoPoint" },
+    country: { type: "Pointer", targetClass: "Country" },
+  },
+  classLevelPermissions: {
+    find: { requiresAuthentication: true },
+    count: { requiresAuthentication: true },
+    get: { requiresAuthentication: true },
+    // Only a user linked into the Admin Parse Role
+    // authorized to manage cities
```
Comment (mtrezza) [inline _includes/defined-schema/getting-started.md]:
```suggestion
    // Only a user linked to the Admin Parse Role is authorized to manage cities
```

**Review by mtrezza — CHANGES_REQUESTED** [review]

**Comment by mtrezza** [comment]
Other than the comment above, is this ready for merge?

**Review by sadortun — APPROVED** [review]
Excellent work ! 🚀

**Comment by Moumouls** [comment]
Ready for merge @mtrezza  !

**Pull request merged**